src/it/override-war-with-snapshot/dependant/src/test/java/org/jenkins_ci/tools/hpi/its/dependant/DependencyTest.java

```diff
@@ -0,0 +1,14 @@
+package org.jenkins_ci.tools.hpi.its.dependant;
+
+import java.io.IOException;
+import org.jenkins_ci.tools.hpi.its.override_war_with_snapshot.Version;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class DependencyTest {
+
+    @Test
+    public void testVersionOfDependency() throws IOException {
+        Assert.assertEquals("2.0.0-SNAPSHOT", Version.getVersion());
```

Comment:
test will fail when run using the dependency declared in the pom, but will pass when using the version of the plugin in the war.